02 copy/app.js

```diff
@@ -0,0 +1,30 @@
+//infO Rozumiem że po prostu wszystko musibyć opakowane tą funkcją init?
```

Comment:
Tak powinno być

Reply:
Hahaha nim to do mnie dotarło, to mój komputer się nasłuchał o kurtyzanach xD

Reply:
CZYTAMY DALEJ vvvvvv----strart----vvvvvv (to tak żebym wiedział na czym skończyłem ;)

- [ ] - kontynuacja

Reply:
O tych sekcjach to czy tałem tutaj:
[html.spec.whatwg.org/multipage/sections.html#the-section-element](https://html.spec.whatwg.org/multipage/sections.html#the-section-element)
- to już też wrzuciłem na slacka